src/test/java/hudson/plugins/git/GitChangeSetUtil.java

```diff
@@ -52,7 +59,7 @@ public static GitChangeSet genChangeSet(boolean authorOrCommitter, boolean useLe
         lines.add("author " + AUTHOR_NAME + " <" + AUTHOR_EMAIL + "> " + AUTHOR_DATE);
         lines.add("committer " + COMMITTER_NAME + " <" + COMMITTER_EMAIL + "> " + COMMITTER_DATE);
         lines.add("");
-        lines.add("    " + COMMIT_TITLE);
+        lines.add("    " + commitTitle);
```

Comment:
Why this change?

Reply:
To make testing easier, it becomes a default params when not using that. I though this could be better than creating a whole new method.